distdir_deps.bzl

```diff
@@ -351,45 +351,49 @@ DIST_DEPS = {
             "remotejdk11_linux_for_testing",
             "openjdk11_linux_archive",
         ],
-        "archive": "zulu11.50.19-ca-jdk11.0.12-linux_x64.tar.gz",
-        "sha256": "b8e8a63b79bc312aa90f3558edbea59e71495ef1a9c340e38900dd28a1c579f3",
-        "strip_prefix": "zulu11.50.19-ca-jdk11.0.12-linux_x64",
+        "archive": "zulu11.52.13-ca-jdk11.0.13-linux_x64.tar.gz",
+        "sha256": "77a126669b26b3a89e0117b0f28cddfcd24fcd7699b2c1d35f921487148b9a9f",
+        "strip_prefix": "zulu11.52.13-ca-jdk11.0.13-linux_x64",
         "urls": [
-            "https://mirror.bazel.build/openjdk/azul-zulu11.50.19-ca-jdk11.0.12/zulu11.50.19-ca-jdk11.0.12-linux_x64.tar.gz",
+            "https://mirror.bazel.build/zulu/bin/zulu11.52.13-ca-jdk11.0.13-linux_x64.tar.gz",
+            "https://cdn.azul.com/zulu/bin/zulu11.52.13-ca-jdk11.0.13-linux_x64.tar.gz",
         ],
         "used_in": ["test_WORKSPACE_files"],
     },
     "remotejdk11_linux_aarch64": {
         "aliases": [
             "remotejdk11_linux_aarch64_for_testing",
         ],
-        "archive": "zulu11.50.19-ca-jdk11.0.12-linux_aarch64.tar.gz",
-        "sha256": "61254688067454d3ccf0ef25993b5dcab7b56c8129e53b73566c28a8dd4d48fb",
-        "strip_prefix": "zulu11.50.19-ca-jdk11.0.12-linux_aarch64",
-        "urls": ["https://mirror.bazel.build/openjdk/azul-zulu11.50.19-ca-jdk11.0.12/zulu11.50.19-ca-jdk11.0.12-linux_aarch64.tar.gz"],
+        "archive": "zulu11.52.13-ca-jdk11.0.13-linux_aarch64.tar.gz",
+        "sha256": "6be187379c26506a4b804b4f734c17e554aebe4204bde58a10b429054cc9cf9f",
+        "strip_prefix": "zulu11.52.13-ca-jdk11.0.13-linux_aarch64",
+        "urls": [
+            "https://mirror.bazel.build/zulu-embedded/bin/zulu11.52.13-ca-jdk11.0.13-linux_aarch64.tar.gz",
+            "https://cdn.azul.com/zulu-embedded/bin/zulu11.52.13-ca-jdk11.0.13-linux_aarch64.tar.gz",
+        ],
         "used_in": ["test_WORKSPACE_files"],
     },
     "remotejdk11_linux_ppc64le": {
         "aliases": [
             "remotejdk11_linux_ppc64le_for_testing",
         ],
-        "sha256": "a417db0295b1f4b538ecbaf7c774f3a177fab9657a665940170936c0eca4e71a",
-        "strip_prefix": "jdk-11.0.7+10",
+        "sha256": "82f14cda71cff99c878bf8400598a87235adb6c81b0337f7077c27e5cac1190c",
+        "strip_prefix": "jdk-11.0.13+8",
         "urls": [
-            "https://mirror.bazel.build/openjdk/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.7+10/OpenJDK11U-jdk_ppc64le_linux_hotspot_11.0.7_10.tar.gz",
-            "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.7+10/OpenJDK11U-jdk_ppc64le_linux_hotspot_11.0.7_10.tar.gz",
+            "https://mirror.bazel.build/github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_ppc64le_linux_hotspot_11.0.13_8.tar.gz",
```

Comment:
Does `%2B8` suppose to be here? I suspect this is breaking the verify_urls test: https://cs.opensource.google/bazel/bazel/+/master:src/test/shell/bazel/verify_workspace.sh;l=45?q=test_verify_urls

Reply:
Replacing `%2B` with `+` could probably work around the test failure, I also confirmed the url will still work.